app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -751,6 +759,26 @@ class HomeFragment : Fragment() {
         lifecycleScope.launch(IO) {
             requireComponents.reviewPromptController.promptReview(requireActivity())
         }
+
+        if (FeatureFlags.showWallpapers) {
+            binding.wordmark.setOnClickListener {
+                val settings = context.settings()
+                val newWallpaper = settings.getWallpaper().switchToNextWallpaper()
```

Comment:
If I understood @MatthewTighe correctly, I thought the idea was to add a function to set the Settings pref inside of the `Wallpaper` class. That way we aren't putting this logic inside Settings or HomeFragment.

Reply:
I didn't added it, as we will be adding the settings dependency to the `Wallpaper` enum, and the function will be switching and storing at the same time, for now the only place that we are going to be using this functionality is here.

Reply:
I guess we could add it, as this is not the permanent solution anyways XD, this is just a placeholder while the functionality evolves

Reply:
@gabrielluong I updated the PR